_includes/parse-server/database.md

```diff
@@ -13,7 +13,7 @@ The Mongo requirements for Parse Server are:
 * MongoDB version 3.6
 * An SSL connection is recommended (but not required).
 
-If this is your first time setting up a production MongoDB instance, we recommend using [ObjectRocket](https://objectrocket.com/). This is a database-as-a-service company which provide fully managed MongoDB instances, and can help you scale up as needed.
+If this is your first time setting up a production MongoDB instance, we recommend using either [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) or [ObjectRocket](https://objectrocket.com/). These are database-as-a-service companies which provide fully managed MongoDB instances, and can help you scale up as needed.
```

Comment:
We should omit trying to describe the companies themselves, because what these companies are is irrelevant for the purpose of the docs, rather than the specific DBaaS service they provide.

I suggest:

> If this is your first time setting up a MongoDB instance, we recommend [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) or [ObjectRocket](https://objectrocket.com/) which provide fully managed MongoDB instances and can help you scale as needed.

Reply:
perfect